docs/developer-docs/setup/playground.md

```diff
@@ -0,0 +1,279 @@
+
+![Add packages](./_attachments/motoko-playground14.png)
+
+From this menu, you can select from the list of pre-imported Vessel packages, or you can input your own from Github.
+
+![Packages](./_attachments/motoko-playground15.png)
+
+Lastly, the 'CANISTERS' portion of the menu showcases your currently running canister(s) and shows the time remaining for the canister's deployment. When this timer runs out, the canister is automatically uninstalled.
+
+To add another canister, select the `+` button next to the 'CANISTERS' option.
+
+![Add canisters](./_attachments/motoko-playground16.png)
+
+From the 'Add a canister' menu, you can import a canister using the canister's ID, as shown below.
+
+![Import by ID](./_attachments/motoko-playground17.png)
+
+Alternatively, you can deploy a canister using the canister's wasm and .did files. 
+
+![Import by wasm](./_attachments/motoko-playground18.png)
+
+Lastly, let's take a look at the two buttons in the top right corner of the Motoko playground UI: the 'Save and & Share' button and the 'Open Example' button.
+
+The 'Save and Share' button can be used to save your canister's code and share it using a live canister URL. Select this button to demonstrate this workflow.
+
+![Save and share](./_attachments/motoko-playground19.png)
+
+In the log, your canister's URL will be returned for you to share with others:
+
+![Canister URL](./_attachments/motoko-playground20.png)
+
+Then, the 'Open Example' button can be used to open another pre-configured example, or import your own from Github.
+
+![Open example](./_attachments/motoko-playground21.png)
+
+This menu will have the same examples and options that the initial welcome menu contained.
+
+![Examples](./_attachments/motoko-playground22.png)
+
+## Running Motoko playground locally
+
+Alternatively, the a version of the Motoko playground can be deployed locally. 
+
+### Prerequisites
+
+- [x] You have an internet connection and access to a shell terminal on your local macOS or Linux computer.
+
+- [x] You have a command line interface (CLI) window open. This window is also referred to as the 'terminal' window.
+
+- [x] You have downloaded and installed the IC SDK package as described in the [installing the IC SDK](./install/index.mdx) page.
+
+- [x] Clone the [Motoko playground repository](https://github.com/dfinity/motoko-playground.git).
+
+- [x] Download and install [npm](https://nodejs.org/en/download/).
+
+- [x] Install `mops` with the command `npm i -g ic-mops`.
+
+- [x] Install [Rust](https://www.rust-lang.org/tools/install).
+
+- [x] Add wasm32 target to Rust with the command `rustup target add wasm32-unknown-unknown`.
+
+Then, run the following commands to deploy the Motoko playground to your local replica:
+
+```
+npm install # Install `npm` dependencies
+npm start # Run the local development server
+dfx deploy
+```
+
+### Creating editor integrations
```

Comment:
```suggestion
## Creating editor integrations
```

That's really cool, didn't know about this one! I think this should be its own H2 section since it doesn't only apply to local deployments